More file formats (#772)
Codecademy/docs: content/general/concepts/file-formats/file-formats.md

|   [PHP (Hypertext Preprocessor)](https://www.codecademy.com/resources/docs/php)   |  `.php`   | Contains a PHP program.                |
|            [Python](https://www.codecademy.com/resources/docs/python)             |   `.py`   | Contains a Python program.             |
|                 [R](https://www.codecademy.com/resources/docs/r)                  |   `.r`    | Contains an R program.                 |
|              [Ruby](https://www.codecademy.com/resources/docs/ruby)               |   `.rb`   | Contains a Ruby program.               |
| [SQL (Structured Query Language)](https://www.codecademy.com/resources/docs/sql)  |  `.sql`   | Contains an SQL program.               |
|             [Swift](https://www.codecademy.com/resources/docs/swift)              | `.swift`  | Contains a Swift program.              |
| [TML (HyperText Markup Language)](https://www.codecademy.com/resources/docs/html) |  `.html`  | Contains an HTML web page.             |
|      [TS (TypeScript)](https://www.codecademy.com/resources/docs/typescript)      |   `.ts`   | Contains a Typescript program.         |

> ### Image file formats

Reviewer: ```suggestion
### Image File Formats
```

|                                       File Format                                       | Extension | Description                                                                                                  |
| :-------------------------------------------------------------------------------------: | :-------: | ------------------------------------------------------------------------------------------------------------ |
| [SVG](https://www.codecademy.com/resources/docs/general/svg) (Scalable Vector Graphics) |  `.svg`   | Great for responsive design. Used for images that can be scaled up or down on a page without losing quality. |
|                         JPEG (Joint Photographic Experts Group)                         |  `.jpeg`  | Standard use for minimized images.                                                                           |
|                             PNG (Portable Network Graphics)                             |  `.png`   | Common file format for high resolution images.                                                               |

> ### Other file formats

Reviewer: ```suggestion
### Other File Formats
```

File formats exist for different types of files. Some common file types and formats are listed below:

### Programing file formats

|                                       File Format                                       | Extension | Description                                                                                                  |
| :-------------------------------------------------------------------------------------: | :-------: | ------------------------------------------------------------------------------------------------------------ |

> | JS (Javascript)                                                                         | **.js**   | Contains a Javasript program.                                                                                |

Reviewer 2: ```suggestion
| JS (JavaScript)                                                                         | **.js**   | Contains a JavaScript program.                                                                                |
```

> | [C++](https://www.codecademy.com/resources/docs/cpp)                                    | **.cpp**  | Conatins a C++ program.                                                                                      |

Reviewer: Instead of being in **bold**, these extensions should be in `back-ticks`.

~**.cpp**~ -> `.cpp`

> | Python                                                                                  | **.py**   | Contrains a Python program.                                                                                  |

Reviewer 2: ```suggestion
| Python                                                                                  | **.py**   | Contains a Python program.                                                                                  |
```

| [CSS(Cascading Style Sheets)](https://www.codecademy.com/resources/docs/css)            | `.css`    | Contains the styling for an HTML page.                                                                       |

> | [Go ](https://www.codecademy.com/resources/docs/go)                                     | `.go`     | Conatins a Go program.                                                                                       |

Reviewer: ```suggestion
| [Go](https://www.codecademy.com/resources/docs/go)                                     | `.go`     | Conatins a Go program.                                                                                       |
```

| [Java](https://www.codecademy.com/resources/docs/java)                                  | `.java`   | Conatins a Java program.                                                                                     |
| [JS (JavaScript)](https://www.codecademy.com/resources/docs/javascript)                 | `.js`     | Contains a JavaScript program.                                                                               |

> | [PHP (Hypertext Preprocessor)  ](https://www.codecademy.com/resources/docs/php)         | `.php`    | Contains a PHP program.                                                                                      |

Reviewer: ```suggestion
| [PHP (Hypertext Preprocessor)](https://www.codecademy.com/resources/docs/php)         | `.php`    | Contains a PHP program.                                                                                      |
```

### Other file formats

|                                       File Format                                       | Extension | Description                                                                                                  |
| :-------------------------------------------------------------------------------------: | :-------: | ------------------------------------------------------------------------------------------------------------ |
| [CSV](https://www.codecademy.com/resources/docs/general/csv) (Comma Separated Values)   | **.csv**  | Contains spreadsheet/table data.                                                                             |
| PDF (Portable Document Format)                                                          | **.pdf**  | Created to be accessed on many devices and operating systems. Contains text and image data.                  |
| MD (Markdown)                                                                           | **.md**   | Contains easily readable markup.                                                                             |

> | Executable                                                                              | **.exe**  | Contains and executable.                                                                                     |

Reviewer 3: ```suggestion
| Executable                                                                              | **.exe**  | Contains an executable.                                                                                     |
```

| TXT (Text)                                                                              | **.txt**  | Contains text.                                                                                               |

> | Zip                                                                                     | **.zip**  | Compresses files into a fold that can be uncompresed into its original form.                                 |

Reviewer 3: ```suggestion
| Zip                                                                                     | **.zip**  | Compresses files into a folder that can be uncompressed into its original form.                                 |
```
